packages/frontend2/src/app/api/(public)/scaling/summary/route.ts

```diff
@@ -0,0 +1,45 @@
+import { NextResponse } from 'next/server'
+import { getScalingSummaryEntries } from '~/server/features/scaling/summary/get-scaling-summary-entries'
+import { getTvlChartData } from '~/server/features/scaling/tvl/utils/get-tvl-chart-data'
+
+export async function GET() {
+  const entries = await getScalingSummaryEntries()
+  const chart = await getTvlChartData({
+    range: '30d',
+    filter: { type: 'layer2' },
+  })
+  return NextResponse.json({
+    success: true,
+    data: {
+      chart: chart.map(([timestamp, canonical, external, native, ethPrice]) => [
+        timestamp,
+        canonical / 100,
+        external / 100,
+        native / 100,
+        ethPrice,
+      ]),
+      projects: entries.reduce<Record<string, unknown>>((acc, entry) => {
+        acc[entry.id] = {
+          ...entry,
+          // Remap change to change7d, remove TVL warnings
+          tvl: {
+            ...entry.tvl,
+            change: undefined,
+            warnings: undefined,
+            change7d: entry.tvl.change,
+          },
+          // For stage - only return the stage name
+          stage: entry.stage.stage,
+          // Strip out the following fields
+          entryType: undefined,
+          href: undefined,
+          isVerified: undefined,
+          hasImplementationChanged: undefined,
+          warning: undefined,
+          showProjectUnderReview: undefined,
+        }
+        return acc
```

Comment:
Let's not spread the entry object if we want to strip smth. In the future someone could add a property to entry and it would be exposed automatically.

Reply:
use Object.fromEntries(Object.entries(entries)) maybe